store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -532,6 +532,7 @@ public DispatchRequest checkMessageAndReturnSize(java.nio.ByteBuffer byteBuffer,
 
             return dispatchRequest;
         } catch (Exception e) {
+            log.error("CommitLog: check message failed", e);
```

Comment:
There is no need to print logs here, because rocketmq master-slave replication is is based on streams rather than messages , so it is common to dispatch half messages  If you print logs here, there will be a lot of dirty logs.